src/app/services/login.service.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpClient, HttpErrorResponse, HttpEvent, HttpHeaders, HttpResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LoginService {
+  tournamentUrl = "https://tournament-backend-yjm8.onrender.com/v1/tournaments";
+  auth_token!: string;
+  
+
+  handleError(error: HttpErrorResponse) {
+    if (error.error instanceof ErrorEvent) {
+      console.log(`An error occurred: ${error.error.message}`);
+    }
+    else {
+      console.error(
+        `Backend returned code ${error.status}, body was: ${error.error}`
+      );
+    }
+    return throwError(() => (new Error("Something went wrong, try again later")));
+  }
+
+
+  constructor(private http: HttpClient) {}
+  
+  authenticate(username: string, password: string): Observable<boolean> {
+    return this.http.post<any>(this.tournamentUrl + "login", {
+      name: username, password: password
```

Comment:
```suggestion
      username: username, password: password
```
![image](https://user-images.githubusercontent.com/26899292/228525628-84324c7d-57f7-4e5a-b1e8-f2ecd6fcbeee.png)